README.md

```diff
@@ -8,6 +8,12 @@ This repo contains Ansible code for OpenShift and Atomic Enterprise.
   ```
     dnf install -y ansible-1.9.4 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
   ```
+  - RHEL 7:
+  ```
+  yum install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  rpm --import http://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
+  yum install ansible1.9-1.9.4-2.el7 rubygem-thor rubygem-parseconfig util-linux pyOpenSSL libffi-devel python-cryptography
```

Comment:
I think the only thing that should be needed here is to install just the ansible package. I can't think of a reason why the python dependencies for ansible wouldn't pull in properly or why we would need the non-python deps at all here.